src/Panel/File.php

```diff
@@ -299,8 +302,7 @@ public function pickerData(array $params = []): array
 			'filename' => $name,
 			'dragText' => $this->dragText('auto', $absolute ?? false),
 			'type'     => $this->model->type(),
-			'url'      => $this->model->url(),
-			'uuid'     => $uuid ?? $id,
```

Comment:
Can you explain what this did before and why it can be removed? Don't fully understand the logic.

Reply:
Me neither - @bastianallgeier needs to answer this, I couldn't find out any meaning from reading the code and it worked without it during my testing. That's why I considered it dead code.

Reply:
There's a dark memory that this was only added for some potential plugins that would need the URL. Don't remember at all why we added this weird uuid thing.

Reply:
We still have a `uuid` entry (now defined in Panel\Model and now containing the new UUID. But maybe that's sufficient for potentially existing plugins?

Reply:
I would remove it. I'm really not sure if it has ever been used. We have new ideas for the pickers anyway and if a plugin has a problem with the removal, we can fix it easily.